openshift/datasync-showcase-no-amq.yml

```diff
@@ -0,0 +1,359 @@
+---
```

Comment:
It is strange pattern. I would expect showcase and showcase-amq. 
Any reason why we do this?

Reply:
Ahh.. The main reason is just so no changes are needed in the integreatly installer. I am happy to change it though!

Reply:
Ah.. But then would other showcase have subscription working without AMQ - probably will need some non AMQ broker to be complete anyway so we practically getting generic showcase and integrately one